src/components/cells/DescriptionCell.vue

```diff
@@ -79,25 +72,23 @@ export default {
   },
 
   computed: {
-    ...mapGetters([
-    ]),
+    ...mapGetters([]),
 
-    content () {
+    content() {
```

Comment:
This rule is not good. It's not coherent with the rest and I'm not sure it works.

Reply:
This rule is forced by Prettier 2.x, no options are available to disable this one, but the JS code is still valid.